portal/config/eproms/QuestionnaireBank.json

```diff
@@ -180,10 +180,10 @@
     },
     {
       "classification": "recurring",
-      "due": "{\"days\": 14}",
-      "expired": "{\"days\": 29}",
+      "due": "{\"months\": 1, \"days\": 14}",
```

Comment:
I don't believe this is what you want.  The docs for this are hidden in the source code, I see.  Most everything is *relative* to `start`.  In fact, I think you may want due to be days=0.

Reply:
Thanks. Hmm, entry.start is currently set to 0 for all... Do you mean relative to entry.recurs.start ? I'll take another swing at this once the windows requirements have settled (hopefully tomorrow)

Reply:
Correct.  Here's the significant code, to determine the "start" of a "recurring QB" (`self`), where `ic` == iteration count, and `trigger_date` is the initial consent:
```
                start = (trigger_date + RelativeDelta(self.start) +
                         RelativeDelta(recur.start) +
                         (ic * RelativeDelta(recur.cycle_length)))
```